plugins/quota/actor.go

```diff
@@ -37,7 +37,7 @@ func isQuotadStopRequired(volumes []*volume.Volinfo) bool {
 	return true
 }
 
-func (actor *quotadActor) Do(v *volume.Volinfo, key, value string, logger log.FieldLogger) error {
+func (actor *quotadActor) Do(v *volume.Volinfo, key, value string, volopt xlator.VolumeOpType, logger log.FieldLogger) error {
```

Comment:
The variable name `volopt` often means 'volume option. Please use `volOp` to clearly indicate that it's a volume operation. Same comment below.

Reply:
done